ParsedownExtra.php

```diff
@@ -250,6 +250,31 @@ protected function blockSetextHeader($Line, array $Block = null)
         return $Block;
     }
 
+    #
+    # Fenced Code
+
+    protected function blockFencedCode($Line)
+    {
+        if (preg_match('/^(['.$Line['text'][0].']{3,}[ ]*([\w-]+)?)([ ]+\{('.$this->regexAttribute.'+)\})?[ ]*$/', $Line['text'], $matches))
```

Comment:
Could we just match anything prior to the regexAttribute? (since we'll check this text is correct with the `parent::blockFencedCode` call). This would fix this issues with test currently failing (since we updated the regex in Parsedown core and added some new test cases).